library/arangodb

```diff
@@ -8,3 +8,5 @@
 3.2: https://github.com/arangodb/arangodb-docker@53829f3067e6191bbc38fb373a35fec615bd0056 stretch/3.2.4
 3.2.4: https://github.com/arangodb/arangodb-docker@53829f3067e6191bbc38fb373a35fec615bd0056 stretch/3.2.4
 latest: https://github.com/arangodb/arangodb-docker@53829f3067e6191bbc38fb373a35fec615bd0056 stretch/3.2.4
+devel: https://github.com/arangodb/arangodb-docker@545283a1d09a428b1eee037f0449554e0a3a8206 stretch/devel
+devel: https://github.com/arangodb/arangodb-docker@545283a1d09a428b1eee037f0449554e0a3a8206 stretch/devel
```

Comment:
@dothebart, it seems your bot script has a bug.  Was is supposed to add this version?